doc/sailfish-webview-webview.qdoc

```diff
@@ -123,26 +612,191 @@
-  \l{WebView}. Setting this value allows the web content to be arranged to
-  compensate for the overlapped area.
+/*!
+  \qmlmethod WebView::loadText(string text, string mimeType)
+  \brief Loads the specified \a text content, displaying appropriately for
+         the specified \a mimeType
+*/
 
-  When using a \l{WebView} this property will be set automatically when the
-  keybaord is opened or closed.
+/*!
+  \qmlmethod WebView::goBack()
+  \brief Navigate back to the previous location in history
 */
 
 /*!
-  \qmlsignal WebView::contentOrientationChanged(Qt::ScreenOrientation orientation)
-  \brief This signal is emitted when the screen orientation of the rendered
-         content changes.
+  \qmlmethod WebView::goForward()
+  \brief Navigate forward to the next location in history
+*/
+
+/*!
+  \qmlmethod WebView::stop()
+  \brief Stop loading the current URL or content
+*/
+
+/*!
+  \qmlmethod WebView::reload()
+  \brief Reload the current URL or content
+*/
+
+/*!
+  \qmlmethod WebView::load(string url)
+  \brief Loads the specified \a url
+*/
+
+/*!
+  \qmlmethod WebView::sendAsyncMessage(string name, variant data)
+  \brief Send an asynchronous message with the specified \a name and \a data
+
+  If a message listener is registered for the message, it will handle it
+  appropriately (e.g. show an appropriate popup, etc).
+
+  \sa addMessageListener()
+*/
+
+/*!
+  \qmlmethod WebView::addMessageListener(string name)
+  \brief Register a listener for asynchronous messages with the
+         specified \a name
+*/
+
+/*!
+  \qmlmethod WebView::loadFrameScript(string name)
+  \brief Loads the specified frame script
+
+  // TODO: What is a frame script?  Why would someone want to load one?
+  // TODO: is the \a name a filename?
+*/
+
+/*!
+  \qmlmethod WebView::newWindow(string url)
+  \brief Opens the specified \a url in a new web view window (tab)
+
+  The new web view window will have its \l{parentId} set to the \l{uniqueId}
+  of this web view instance.
+
+  // TODO: is this correct?  What is "window" in this sense?
 */
 
 /*!
   \internal
+  \qmlmethod WebView::synthTouchBegin(variant touches)
+  \brief Synthesize a touch-begin event
 
-  \qmlsignal WebView::openUrlInNewWindow()
-  \brief This signal is emitted to indicate a request to open a new window.
+  \sa useQmlMouse, synthTouchMove(), synthTouchEnd()
+*/
+
+/*!
+  \internal
+  \qmlmethod WebView::synthTouchMove(variant touches)
+  \brief Synthesize a touch-move event
+
+  \sa useQmlMouse, synthTouchBegin(), synthTouchEnd()
+*/
+
+/*!
+  \internal
+  \qmlmethod WebView::synthTouchEnd(variant touches)
+  \brief Synthesize a touch-end event
+
+  \sa useQmlMouse, synthTouchBegin(), synthTouchMove()
+*/
+
+/*!
+  \qmlmethod WebView::suspendView()
+  \brief Suspend rendering of the view
+
+  \sa resumeView()
+*/
+
+/*!
+  \qmlmethod WebView::resumeView()
+  \brief Resume rendering of the view
+
+  \sa suspendView()
+*/
+
+/*!
+  \qmlmethod WebView::recvMouseMove(int posX, int posY)
```

Comment:
internal

Reply:
can be removed the docs... see JB#54343 / JOLLA-159.

Reply:
removed